sqlmesh/core/engine_adapter/base.py

```diff
@@ -168,13 +168,12 @@ def create_index(
         """
         if not self.SUPPORTS_INDEXES:
             return
+
         expression = exp.Create(
             this=exp.Index(
                 this=exp.to_identifier(index_name),
                 table=exp.to_table(table_name),
-                columns=exp.Tuple(
-                    expressions=[exp.to_column(c) for c in columns],
-                ),
+                columns=[exp.to_column(c) for c in columns],
```

Comment:
@tobymao can you take a look here? Seems like this became invalid due to https://github.com/tobymao/sqlglot/commit/fbf5f472fffa79538d1a58bfd1c1562796bafc4d.

Note that by assigning a list of `Column`s we'll generate a different AST here than what `parse_one` would give:
```python
>>> import sqlglot
>>> sqlglot.parse_one("CREATE INDEX my_idx ON tbl (a, b)")
(CREATE this:
  (INDEX this:
    (IDENTIFIER this: my_idx, quoted: False), table:
    (TABLE this:
      (IDENTIFIER this: tbl, quoted: False)), columns:
    (ORDERED this:
      (COLUMN this:
        (IDENTIFIER this: a, quoted: False)), desc: False, nulls_first: True),
    (ORDERED this:
      (COLUMN this:
        (IDENTIFIER this: b, quoted: False)), desc: False, nulls_first: True)), kind: INDEX)
```

Reply:
yea that was wrong

Reply:
columns is fine

Reply:
Cool, thanks for verifying. Doing a final pass on the codebase to see if there's any other place where we could have a regression, but it's probably good to go.